products/sourcegraph.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Sourcegraph
+category: server-app
+tags: java-runtime
+permalink: /sourcegraph
+alternate_urls:
+- /sourcegraph-code-search
```

Comment:
Not sure if this is helpful. We typically provide alternate_urls for urls that users might type and expect to work (k8s, eks etc). `/sourcegraph` is already a great URL, so don't see a need here. We could maybe add `/sg`, which seems to be used in a few places, but that is too ambiguous.

```suggestion
```